### ISSUE TITLE
Stuck at `Loading QR` step after attempting to create 2nd headless session

### ISSUE THREAD

**Issue by HypedMind** [smashah/sulla#125]
**Describe the bug**
Possible duplicate of #120 , just want to provide my sample. 

After the app started, the first time was able to load QR code and created valid HEADLESS session after scan the qr code. After this, when attempting to create another HEADLESS session again, this time, it stucked at the step `Loading QR`, without mentioning tos blocked or other reason, just stuck there.

**Steps to Reproduce**
1. Create 1st session.
2. Wait for it to load valid session.
3. Create 2nd session.
4. It was stuck at the step `Loading QR`.

**create() code**
This is the code you use to create the client. e.g
```javascript
const sulla = require('sulla-hotfix');

var executablePath = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';

sulla.create("session_1", {
    executablePath: executablePath,
    headless: true,
    autoRefresh: true,
    qrRefreshS: 20
}).then(client => start(client));

function start(client) {
    console.log("start");

    client.onMessage(message => {
        console.log(message); 
    });

    sulla.create("session_2", {
        executablePath: executablePath,
        headless: true,
        autoRefresh: true,
        qrRefreshS: 20
    }).then(client => start(client));

    function start(client) {
        console.log("start");

    }
}
```

**Expected behavior**
Expected the 2nd time will be loading QR as the first time.

**DEBUG INFO**
This is the info printed to the console when you start your app. It should look like this
```
Debug Info { WA_VERSION: '0.4.613',
  PAGE_UA: 'WhatsApp/0.4.613 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36',
  SULLA_HOTFIX_VERSION: '1.7.13',
  BROWSER_VERSION: 'HeadlessChrome/79.0.3945.130' 
}
```

**Screenshots**
![Screenshot 2020-02-16 at 10 16 56 AM](https://user-images.githubusercontent.com/36067431/74598108-4de70380-50a6-11ea-81a1-2566f099d011.png)

**Host (please complete the following information):**
 - OS: MacOS & Ubuntu 18 LTS

**Comment by smashah** [comment]
@HypedMind thanks for filling in the template. I will be looking into this soon.

**Comment by smashah** [comment]
@HypedMind fix should be live in v1.7.23
